django_amazon_ses.py

```diff
@@ -39,6 +41,7 @@ def __init__(
         access_key_id = getattr(settings, "AWS_ACCESS_KEY_ID", None)
         secret_access_key = getattr(settings, "AWS_SECRET_ACCESS_KEY", None)
         region_name = getattr(settings, "AWS_DEFAULT_REGION", "us-east-1")
+        session_token = None
```

Comment:
Since a session is a temporary credential, it doesn't make sense to support it as a configuration option from settings, so it defaults to `None`.